crates/sui-framework/packages/sui-framework/tests/random_tests.move

```diff
@@ -51,7 +51,8 @@ module sui::random_tests {
             scenario.ctx(),
         );
         scenario.next_tx(@0x0);
-        let gen1 = random_state.new_generator(scenario.ctx());
+        let mut ctx = tx_context::new_from_hint(@0x0, 1, 0, 0, 0);
```

Comment:
Note the changes here: it was previously using the fact that the object IDs derived from the context were stable which is no longer the case so we manually stabilize them here.

Reply:
I don't fully understand, what happens if this isn't stable?